Database/init-constraints.sql

```diff
@@ -45,25 +45,13 @@ ALTER TABLE events.parameter ADD CONSTRAINT chk_parameter_value CHECK (value ~ '
 
 ALTER TABLE events.parameter ADD CONSTRAINT chk_parameter_key CHECK (key ~ '^[[:alpha:]]+(?:\s[[:alpha:]]+)*$');
 
-ALTER TABLE users.role ADD CONSTRAINT chk_three_roles CHECK (id <= 3);
-
-ALTER TABLE users.role ADD CONSTRAINT chk_role_name CHECK (name ~ '^(Organizer|Admin|User)$');
-
-ALTER TABLE common.status ADD CONSTRAINT chk_five_statuses CHECK (id <= 5);
-
-ALTER TABLE common.status ADD CONSTRAINT chk_status_name  CHECK (name ~ '^(Interested|Participating|Completed|In progress|Waiting for approval)$');
-
-ALTER TABLE common.object_type ADD CONSTRAINT chk_two_object_types CHECK (id <= 2);
-
-ALTER TABLE common.object_type ADD CONSTRAINT chk_object_type_name CHECK (name ~ '^(Event|User)$');
-
-ALTER TABLE common.status
-ADD CONSTRAINT chk_object_type_for_status
-CHECK (
-    (name IN ('Interested', 'Participating') AND object_type_id = 2) OR
-    (name IN ('Completed', 'In progress', 'Waiting for approval') AND object_type_id = 1)
-);
+-- ALTER TABLE common.status
+-- ADD CONSTRAINT chk_object_type_for_status
+-- CHECK (
+--     (name IN ('Interested', 'Participating') AND object_type = 'User') OR
+--     (name IN ('Completed', 'In progress', 'Waiting for approval') AND object_type = 'Event')
+-- );
```

Comment:
This one we can do directly on events.event and users.user, add check where it's status will be in X. Just like it was but not for status table but for each status column directly check the values of enum.